src/Bundle/Command/DebugGridCommand.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Command;
+
+use Sylius\Component\Grid\Provider\GridProviderInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Dumper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+#[AsCommand(name: 'sylius:debug:grid')]
+final class DebugGridCommand extends Command
+{
+    public function __construct(
+        private readonly GridProviderInterface $gridProvider,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument(name: 'grid', mode: InputArgument::REQUIRED, description: 'The grid name or FQCN')
+            ->setHelp("The <info>%command.name% 'App\Grid\DummyGrid'</info> command dumps the definition for the dummy grid.")
```

Comment:
```suggestion
```suggestion
            ->setHelp(<<<'EOF'
The <info>%command.name%</info> command displays all configured grids:

  <info>php %command.full_name%</info>
  
To get specific grid, specify its name (or FQCN):

  <info>php %command.full_name% sylius_product</info>
  
  <info>php %command.full_name% App\Grid\SupplierGrid</info>
EOF
            )
        ;
```
```